addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,28 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("reply action functionality in Chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    pyEnv["mail.message"].create({
+        body: "Test message from B",
+        subtype_id: pyEnv["mail.message.subtype"].search([
+            ["subtype_xmlid", "=", "mail.mt_note"],
+        ])[0],
+        model: "res.partner",
+        res_id: serverState.partnerId,
+        author_id: partnerBId,
+    });
+    await start();
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message-actions button[title='Reply']");
+    await contains("button.active", { text: "Log note" });
+    await contains(".o-mail-Composer-coreHeader", { text: "Replying to Partner B", count: 0 });
+    await contains(".o-mail-Composer-input:focus", { value: "@Partner B " });
+    await click(".o-mail-Composer-send");
```

Comment:
Prefer click on button with label / title that end-users actually see on UI. The classname is not visible on UI so this doesn't feel intuitive enough from just looking at the UI.